<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>dsone</class>
 <widget class="QDialog" name="dsone">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>806</width>
    <height>479</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:0, stop:0 rgba(138, 171, 202, 255), stop:1 rgba(255, 255, 255, 255));</string>
  </property>
  <widget class="QWidget" name="verticalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>100</y>
     <width>411</width>
     <height>311</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_2">
    <item>
     <widget class="QComboBox" name="comboBox_Array">
      <property name="styleSheet">
       <string notr="true">background-color: rgb(237, 255, 153);</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QComboBox" name="comboBox_List">
      <property name="styleSheet">
       <string notr="true">background-color: rgb(237, 255, 153);</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QComboBox" name="comboBox_Stack">
      <property name="styleSheet">
       <string notr="true">background-color: rgb(237, 255, 153);</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QComboBox" name="comboBox_Queue">
      <property name="styleSheet">
       <string notr="true">background-color: rgb(237, 255, 153);</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QComboBox" name="comboBox_BST">
      <property name="styleSheet">
       <string notr="true">background-color: rgb(237, 255, 153);</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QComboBox" name="comboBox_Graph">
      <property name="styleSheet">
       <string notr="true">background-color: rgb(237, 255, 153);</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>640</x>
     <y>20</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Back</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
